Packs/MicrosoftExchangeOnPremise/Integrations/EWSv2/EWSv2_test.py

```diff
@@ -664,3 +664,48 @@ def test_get_time_zone(mocker):
     mocker.patch.object(demisto, 'callingContext', new={'context': {'User': {'timeZone': 'Asia/Jerusalem'}}})
     results = get_time_zone()
     assert results.key == 'Asia/Jerusalem'
+
+
+def test_get_message_for_body_type_no_body_type_with_html_body():
+    body = "This is a plain text body"
+    html_body = "<p>This is an HTML body</p>"
+    result = get_message_for_body_type(body, None, html_body)
```

Comment:
isn't it worth a `pytest.mark.parametrize`?

Reply:
Anar doesn't like parameterize. so nope :)